src/themes/Theme.ts

```diff
@@ -0,0 +1,14 @@
+interface Theme {
+    primaryFillColor: string
```

Comment:
The variable names in Twist-Web can be considered the "gold standard" when it comes to naming and they're aligned with Design. We should use the same ones here (e.g. `--content-primary`, `--chromatic-fill-blue`, ...)